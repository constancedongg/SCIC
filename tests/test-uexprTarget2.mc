|'{g} = 1000.0 '{kg}|;
|'{mm} = 10.0 '{cm}|;

float '{mm/g} x;
float '{cm/kg} y;

int func main() {
    printl("in main");
    x = 9.978;
    printf(x);
    y = x - 5.973;
    printf(y);
    float '{m/kg} z = (x - 5.974) * 0.1;
    printf(z);
    float '{m/kg} w = y - z;
    printf(w);
    return 0;
}