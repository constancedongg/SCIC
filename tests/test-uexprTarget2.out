in main
9.978
400.5
0.4004
3.6046
